packages/sys-apps/cpuid/cpuid-20201006.exheres-0

```diff
@@ -17,12 +17,19 @@ MYOPTIONS="parts: binaries documentation"
 
```

Comment:
Using `parts` for 2-file package seems redundant. Allow me to remove them?

Reply:
If there is a burden to maintain it - maybe worth to remove. Right now it seems that it still works.

I had impression that parts supposed to allow building minimal system (e.g. disabling documentation parts).  And that's probably I added it here.